docs/testing-overview.md

```diff
@@ -352,7 +352,7 @@ Code coverage is measured for each PR using the [codecov.io](https://codecov.io/
 If you're using the built-in [local environment](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md#local-environment), you can run the e2e tests locally using this command:
 
 ```bash
-npm run test-e2e
+WP_BASE_URL=http://example.local WP_USERNAME=admin WP_PASSWORD=password npm run test-e2e
```

Comment:
This won't work according to the previous sentence. I think we should keep this as is and add a new paragraph bellow, something like this:

`If you're using a different setup, you can provide the base URL, username and password like this:`